workers/reporter/src/main/kotlin/reporter/SourceCodeBundleReporter.kt

```diff
@@ -54,12 +57,38 @@ import org.slf4j.LoggerFactory
 const val SOURCE_BUNDLE_FILE_NAME = "source-bundle-archive.zip"
 const val SOURCE_BUNDLE_SUB_DIR = "source_code_bundle"
 
+data class SourceCodeBundleReporterConfig(
+    /**
+     * The license classifications to consider when including packages in the source code bundle.
```

Comment:
Nit: Here you say "classifications" but the property's name contains "categories", so maybe better use that term here as well?

Reply:
Yes, and BTW, I don't why we decided to call the ORT config file "license classifications" but the groups inside "categories" instead of "classes", seems unnecessarily inconsistent.